.github/workflows/ci.yml

```
@@ -0,0 +1,31 @@
+name: Continuous Integration
+on: [push]
+
+jobs:
+  linter:
+    name: Code style
+    runs-on: ubuntu-latest
```

```suggestion
    runs-on: ubuntu-18.04
```

to be coherent with the `tests` below?

and using a fixed version for a dev/CI dependency is a best practice most of the time, that's why I suggest using `18.04` instead of `latest`

---

I agree.

I will also update it to the latest LTS (20.04).